gradle-plugin/kubernetes/src/test/java/org/eclipse/jkube/gradle/plugin/KubernetesPluginTest.java

```diff
@@ -90,4 +92,15 @@ void getTaskPrecedence_withValidProject_shouldReturnTaskPrecedence() {
         .containsEntry("k8sHelmPush", Collections.singletonList(KubernetesHelmTask.class))
         .containsEntry("k8sHelmLint", Collections.singletonList(KubernetesHelmTask.class));
   }
+
+  @Test
+  void register_shouldRegisterTasks_WithoutReturningTask_kubPlugin() {
+    AbstractJKubePlugin<KubernetesExtension> jkubePlugin = mock(KubernetesPlugin.class, RETURNS_DEEP_STUBS);
```

Comment:
Do we need to create mocks with deep stubbing here?

Reply:
You are correct, that's not needed, will do the needful